tests/client/test_client.py

```diff
@@ -156,6 +155,23 @@ def test_lit_references(self):
         response = client.lit.references(self.db_keys.assessment_client)
         assert isinstance(response, pd.DataFrame)
 
+    def test_lit_update_reference(self):
+        client = HawcClient(self.live_server_url)
+        client.authenticate("pm@pm.com", "pw")
+        updated_title = "client test"
+        response = client.lit.update_reference(self.db_keys.reference_linked, title=updated_title)
+        assert isinstance(response, dict)
+        assert response["title"] == updated_title
+
+    def test_lit_delete_reference(self):
+        client = HawcClient(self.live_server_url)
+        client.authenticate("pm@pm.com", "pw")
+        try:
```

Comment:
just saw this test. Might be better to retrieve and make sure response is 200, then delete and assert that the response is None or empty or something?  And then maybe use the retrieve again and assert it returns a 404